doc/guides/networking/overview.rst

```diff
@@ -101,6 +100,11 @@ can be disabled if not needed.
 * **Time Sensitive Networking.** The gPTP (generalized Precision Time Protocol)
   is supported. See :ref:`gptp_interface` for more details.
 
+* **Network shell.** The network shell provides helpers for figuring out
+  network status, enabling/disabling features, and issuing commands like ping
+  or DNS resolving. The net-shell is useful when developing network software.
+  See :ref:`net_shell` for more details.
```

Comment:
It looks like the doc label for this is ``:ref:`shell_label` ``  (which would link to https://docs.zephyrproject.org/latest/reference/shell/index.html)

Reply:
Actually I was waiting #13679 that will introduce the net_shell label.